vignettes/poisson.Rmd

```diff
@@ -0,0 +1,359 @@
+
+```{r}
+expctd <- pdf(p_const, 0:6)
+cbind("observed" = obsrvd, "expected" = expctd)
+```
+
+By and large, all observed and expected frequencies are rather close.
+However, it is not reasonable that all teams score goals with the same probabilities,
+which would imply that winning or losing could just be attributed to "luck" or
+"random variation" alone. Therefore, while a certain level of randomness will
+certainly remain, we should also consider that there are stronger and weaker
+teams in the tournament.
+
+
+## Poisson regression and probabilistic forecasting
+
+To account for different expected performances from the teams in the 2018 FIFA World Cup,
+the `FIFA2018` data provides an estimated `logability` for each team. These
+have been estimated by @Zeileis+Leitner+Hornik:2018 prior to the start of the
+tournament (2018-05-20) based on quoted odds from 26 online bookmakers using
+the bookmaker consensus model of @Leitner+Zeileis+Hornik:2010. The `difference` in
+`logability` between a team and its opponent is a useful predictor for the
+number of `goals` scored.
+
+Consequently, we fit a generalized linear model (GLM) to the data that links
+the expected number of goals per team/match $\lambda_i$ to the linear predictor
+$x_i^\top \beta$ with regressor vector $x_i^\top = (1, \mathtt{difference}_i)$
+and corresponding coefficient vector $\beta$ using a log-link:
+$\log(\lambda_i) = x_i^\top \beta$. The maximum likelihood estimator $\hat \beta$
+with corresponding inference, predictions, residuals, etc. can be obtained in R
+using the `glm()` function with `family = poisson`:
+
+```{r}
+m <- glm(goals ~ difference, data = FIFA2018, family = poisson)
+summary(m)
+```
+
+Both parameters have intuitive interpretations. First, the intercept
+corresponds to the expected log-goals per team in a match of two equally strong teams,
+i.e., with zero difference in log-abilities. The corresponding prediction for the
+number of goals can either be obtained manually from the extracted `coef()`
+or equivalently using the `predict()` function:
+
+```{r}
+lambda_zero <- exp(coef(m)[1])
+lambda_zero
+predict(m, newdata = data.frame(difference = 0), type = "response")
+```
+
+As above, we can also set up a `Poisson()` distribution object and obtain the
+associated expected probability distribution for zero to six goals:
+
+```{r}
+p_zero <- Poisson(lambda = lambda_zero)
+pdf(p_zero, 0:6)
+```
+
+Second, the slope of $`r round(coef(m)[2], digits = 3)`$ can be interpreted
+as an ability elasticity of the number of goals scored. This is because the
+difference of the log-abilities can also be understood as the log of the ability
+ratio. Thus, when the ability ratio increases by $1$ percent, the expected
+number of goals increases approximately by $`r round(coef(m)[2], digits = 3)`$
+percent.
+
+This yields a different predicted Poisson distribution for each team/match
+in the tournament. We can easily set up the vector of all $128$ `Poisson()`
```

Comment:
```suggestion
in the tournament. We can set up the vector of all $128$ `Poisson()`
```

Reply:
I incorporated your suggestion before I realized that I could have just hit the "Commit suggestion" button here. (Sorry, I'm still relatively new to GitHub...)